cockpit/src/main/java/com/polidea/cockpit/paramsedition/viewholder/NumberParamViewHolder.kt

```diff
@@ -0,0 +1,32 @@
+package com.polidea.cockpit.paramsedition.viewholder
+
+import android.text.Editable
+import android.text.TextWatcher
+import android.view.View
+import android.widget.EditText
+import com.polidea.cockpit.R
+import com.polidea.cockpit.core.CockpitParam
+
+abstract class NumberParamViewHolder<T : Number>(view: View) : ParamBaseViewHolder<T>(view) {
+
+    private val value: EditText = view.findViewById(R.id.cockpit_number_param_value)
+
+    init {
+        value.addTextChangedListener(object : TextWatcher {
```

Comment:
Maybe we can implement our own CockpitTextWatcher, so inside init block we'd only need to override onTextChanged?

Reply:
I would rather do something similar to Java's MouseListener interface and MouseAdapter class which implements this listener allowing us to override specific methods only. 
Following this convention we can create TextWatcherAdapter